src/components/Pivot/Pivot.scss

```diff
@@ -123,6 +123,14 @@
 
     &.is-selected {
       font-weight: $ms-font-weight-semilight;
+
+      .ms-Pivot-count {
+        font-weight: $ms-font-weight-regular;
+      }
+    }
+
+    .ms-Pivot-count {
+      @include margin-left(5px);
```

Comment:
Also changing the font-weight on selection risks horitzontal jitters. Please ensure that on changing tabs, there are absolutely no jitters to the other tabs.